uk.co.compucorp.civicrm.hremails/email-templates/dist/sickness-request.civicrm.html

```diff
@@ -202,7 +202,11 @@ <h2 class="callout-title" style="Margin: 0; Margin-bottom: 0; color: #4D5663; fo
                           {foreach from=$sicknessRequiredDocuments item=value key=id}
                             <div class="request-document" style="Margin-bottom: 10px; margin-bottom: 10px;">
                               <span class="request-document-required" style="Margin-right: 5px; margin-right: 5px; width: 25px;">
-                                <input type="checkbox" disabled {if="" in_array($id,="" $leaverequireddocuments)}checked{="" if}="" style="Margin: 0; margin: 0;">
```

Comment:
I see `$leaverequireddocuments` is lowercase for some reason, could it be a reason for the error?

The code provided has a lot of unnecessary repetition. Is there a reason why {if in_array($id, $leaveRequiredDocuments)}checked{/if} would not work?